wouso/games/teamquest/templates/teamquest/teamhub.html

```diff
@@ -0,0 +1,156 @@
+{% extends 'interface/section_base.html' %}
+{% load i18n %}
+
+{% block title %}{% trans 'Team Hub' %}{% endblock %}
+
+{% block scripts %}
+    <script type="text/javascript" src="{{ basepath }}/static/js/tabs.js"></script>
+    <link rel="stylesheet" type="text/css" href="{{ basepath }}/static/css/tabs.css" />
+{% endblock %}
+
+{% block sectiontitle %}{% trans 'Team Hub' %}{% endblock %}
+{% block sectioncontent %}
+    
+    <ul class="tabs">
+        {% if group %}
+            <li>
+                <a href="#team" id="group-click" class="no-redir">
+                    {% trans 'Team' %}
+                </a>
+            </li>
+        {% else %}
+            <li>
+                <a href="#find_team" id="create_group-click" class="no-redir">
+                    {% trans 'Teams' %}
+                </a>
+            </li>
+        {% endif %}                 
+        <li>
+            <a href="#invitations" id="invitations-click" class="no-redir">
+                {% trans 'Invitations' %}
+            </a>
+        </li>
+        <li>
+            <a href="#notifications" id="group-click" class="no-redir">
+                {% trans 'Notifications' %}
+            </a>
+        </li>
+    </ul>
+    <div class="spacer"></div>
+    <div class="tab_container">
+        {% if group %}
+        <div id="team" class="tab_content">
+            <div class="spacer"></div>
+            <a class="btn btn-info pull-right" href="{% url setup_leave %}" onclick="return confirm('Do you really want to abandon team {{group.name}}?');"> {% trans 'Leave team' %} </a>
+            <h1> {{ group.name }} </h1>
+            {% for player in group.users.all %}
+                <p>
+                {% if player.is_group_owner %}
+                    <strong> {{ player }} </strong> {% trans '[owner]' %}
+                {% else %}
+                    <em> {{ player }} </em>
+                {% endif %}
+                {% if ownership and not player.is_group_owner %}
+                     <a class="btn btn-info" href="{% url setup_kick player.id %}" onclick="return confirm('Do you really want to banish player {{player.nickname}}?');"> {% trans 'Banish' %} </a>
```

Comment:
We only use bootstrap for Control Panel. The btn, btn-info classes have no effect here.